### ISSUE TITLE
Highcharter is NOT automatically resize output in shinydashboard

### ISSUE THREAD

**Issue by chenx2018** [jbkunst/highcharter#628]
The **develop-version** of highcharter has the following issue:

highchartOutput **DOES NOT automatically resize the width** of the plot **when hiding the sidebar** in shinydashboard. The width of the plot always remains same. 

I provide an example here,

```
library(shinydashboard)
library(shiny)
library(highcharter)
library(dplyr)

ui <- dashboardPage(
  dashboardHeader(),
  dashboardSidebar(),
  dashboardBody(
    box(
      width = 12,
      highchartOutput("plot")
    )
  )
)

server <- function(input, output) {
  
  output$plot <- renderHighchart({
    highcharts_demo()
  })
}

shinyApp(ui, server)
```

**Comment by chenx2018** [comment]
This issue is related to #603 and #298. I tried 

`$(document).on("click", ".sidebar-toggle", function() { $(window).trigger("resize"); });`

BUT it still does not work for development-version highcharter when I use renderHighchart & highchartOutput functions. 

However, if I found renderHighchart2 & highchartOutput2 working well for resizing. I am not sure this would help you @henry090. 

One more thing, since renderHighchart2 & highchartOutput2 does not support heatmap, I still need to use renderHighchart & highchartOutput to solve the problem. ANY suggestions? @jbkunst

**Comment by chenx2018** [comment]
solved by fix the highcharter.js file.

 ```
/* fix bug here */
  resize: function(el, width, height, instance) {
    
    /* http://stackoverflow.com/questions/18445784/ */
    var chart = $("#" +el.id).highcharts();
    var height = chart.renderTo.clientHeight; 
    var width = chart.renderTo.clientWidth; 
    chart.setSize(width, height); 

  }
```

**Comment by jbkunst** [comment]
Hi @chenx2018 

Sorry for my _inactivity time_ but now I'm back.

Thanks for notice this. This is problem came up with a previous fix #534. 

Due the reflow option is missing, the `if` statement (`chart && chart.options.chart.reflow === true`) don't  return a `true`. So I put the explicity reflow true in the default option. 

https://github.com/jbkunst/highcharter/commit/2101848e99e7c4330db251ad081aa5998cd987ae#diff-e549505eb95036528ca3b125f62915a6R72

Please, tell me if your apps work as expected.

Kind regards.
